### PR TITLE
Add string template splitting section 

### PR DESCRIPTION
**Pull request by poorna2152**
## Purpose
Closes #https://github.com/ballerina-platform/ballerina-dev-website/issues/9061

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/backtick-templates/backtick_templates.out [hunk 1]
```diff
@@ -1,3 +1,4 @@
 $ bal run backtick_templates.bal
 Hello, James
 Backtick:`
+A string-template-expr is evaluated by evaluating the expression in each interpolation in the order in which they occur.
```
Comment (gimantha) [inline examples/backtick-templates/backtick_templates.out:4]: Seems the string is not split into a new line

Comment (gimantha) [reply examples/backtick-templates/backtick_templates.out:4]: ignore the comment please.

Comment (gimantha) [reply examples/backtick-templates/backtick_templates.out:4]: On a second thought, I feel having newline also in a middle of the string template and showing the output is also worth, as it is useful for examples like email content..

File: examples/backtick-templates/backtick_templates.bal [hunk 2]
```diff
@@ -10,4 +10,10 @@ public function main() {
     // Concatenates `Backtick:` strings with `.
     string s2 = string `Backtick:${"`"}`;
     io:println(s2);
+
+    // If required, a single-line string can be split into a multiline string template by breaking
```
Comment (MaryamZi) [inline examples/backtick-templates/backtick_templates.bal:14]:
```suggestion
    // If required, a single-line string template can be split into a multiline string template by breaking
```

**Review by MaryamZi — DISMISSED** [review]

File: examples/backtick-templates/backtick_templates.out (hunk 1 above)
Comment (poorna2152) [reply examples/backtick-templates/backtick_templates.out:4]: The string `s2` here, https://ballerina.io/learn/by-example/backtick-templates/. Isn't it kind of similar to using special characters like new line in a string template.

**Review by gimantha — APPROVED** [review]

**Pull request merged**